### ISSUE TITLE
Guideline for Irlen Syndrome

### ISSUE THREAD

**Issue by alastc** [w3c/wcag#1274]
Noted by @awkawk, Irlen Syndrome causes issues with reading from screens:
https://twitter.com/DannyBittman/status/1292538843594395649

Possibly more on the user-agent side than the authored content, but there might be things for authors to do/avoid.

**Comment by patrickhlauke** [comment]
reduced contrast, which may be in direct opposition to the requirement from other user groups for contrasts of 4.5:1 / 3:1 or above? this is where things will start to get tricky, particularly without an appropriate media query that could cover this too...

**Comment by JAWS-test** [comment]
I don't think it's that complicated. The UAAG have been demanding for many years that individual colour settings for text be possible in the browser (https://www.w3.org/TR/UAAG20/#gl-text-config). This would have to be extended to colour settings for non-text content.

> The user can globally set all of the following characteristics of visually rendered text content: ... Text color and background color, choosing from all platform color options
 
In addition, the WCAG should introduce a SC that websites should not prevent individual colour adjustments. Analogous to 1.4.12 Text Spacing. 

Alternatively, the existing criteria should be used to check for  colour adjustments. Suitable for this purpose are e.g. 1.3.1 and SC 1.4.5 with regard to images of text. Unfortunately in my opinion there is no agreement to check after SC 1.3.1 whether websites support Windows contrast adjustment. The Windows contrast adjustment also allows to set low contrasts.

A third possibility would be to make the first paragraph of SC 1.4.8 an AA criterion and not limit it to text

> Foreground and background colors can be selected by the user.

**Comment by StommePoes** [comment]
I would much rather all OSes offered this functionality, and then if not them, all web user agents.
Asking developers to include contrast widgets is asking for lots of pain for the users as devs get it wrong or their managers/PMs decide it's an extra cost with no return. Adding Yet More Media Queries is something I personally would love to avoid, esp since these can even *override* user settings simply because a Media check returns as "true" (like the forced-colours settings).
This allows devs to stick to sufficient minimum text contrast from 2.0 and everything else can be controlled by the user on the OS end. I've only been "pro" contrast-lowering widgets on web pages specifically because setting up low contrast settings on the user side currently is difficult or impossible.

**Comment by alastc** [comment]
According to the [BMJ](https://www.bmj.com/content/349/bmj.g4872/rr/761729):

> there are more than 100 scientific research studies on the topic of Irlen Syndrome that are published in peer-reviewed scientific journals. This research has established a hereditary component of the disorder, a number of biochemical markers for problems associated with Irlen Syndrome, and differences between both the anatomy and functioning of brains of individuals with Irlen Syndrome.

(Citations removed for brevity, but see the link for the listing.)

Interestingly they note that it isn't an optometry or medicine thing, but more of a cognitive thing: 

> As a perceptual problem, it is similar to other processing problems (both visual and auditory) that are diagnosed by psycho-educational testing and treated within the educational system. 

I think that is sufficient to assume it is a disability related issue that people experience. 

Whether or how that is covered by WCAG is another question, and it does seem to be something best dealt with on the user-agent end.

**Comment by Myndex** [comment]
Hi Alastair @alastc 

> According to the [BMJ](https://www.bmj.com/content/349/bmj.g4872/rr/761729):
> > _there are more than 100 scientific research studies on the topic of Irlen Syndrome that are published in peer-reviewed scientific journals. ...._

That cite was on the BMJ, also known as _the "the journal the researcher wanted wouldn't publish this, so they asked us to"_ journal. And the author of that short piece (that is mainly a listing of supporting papers) is herself an employee of the Irlen Institute.

All I am saying is that there is some controversy and a lack of consensus on that subject. Morgellons is another one, that a filmmaker friend of mine did a documentary on, wherein the medical establishment mostly rejects it, though there are people with *some* condition believing it to be Morgellons. 

And I am not saying that Irlen is at the level of Morgellons. But when it comes to "ailments" it is all too easy for us, as humans, to hear that some set of symptoms is "X thing" and then decide that's what the issue is. And not just patients. Doctors have this problem when diagnosing their patients, and such is the nature of this beast.

### _Well, Golly, Peer-Reviewed_
And this is why I chuckle regarding the aggressive _"it has to be peer reviewed"_ clamoring I've dealt with regarding in-progress, ongoing research. 

_**Peer review does not mean something is consensus nor even close to it.**_

Peer review itself was not even a part of the scientific methodology until late in the 20th century. As an FYI, of the 300 papers Albert Einstein published, only one was actually "peer reviewed". Today the reality is, with the mass of pay-to-play open access journals available, there is no shortage of "peer reviewed" anything, and that generally is not much in the way of actual proof.

Nearly all of the "open" journals charge the author exorbitant fees to publish the paper _(no, the researcher does not get paid, THEY have to pay, if any one wonders what is wrong with our world, this is one of the "wrong things")._ The "open" version of Nature charges $11,000 US to publish a paper. 😳 The end result is that any corporate interest that want to can have whatever point of view they want published in a "peer reviewed journal" all that's needed is a bare modicum of science value and some indication of due diligence.

That said, the link to the peer reviewed article I provided in my post above was a review article critically examining the earlier peer-reviewed papers such as in the article you linked, which I initially found on [NIH](https://pubmed.ncbi.nlm.nih.gov/30970133/)

Here's another: [Irlen lenses: a critical appraisal](https://pubmed.ncbi.nlm.nih.gov/2136527/)

### Abstract _(Emphasis Mine)_
> _The purpose of this paper is to assess the credibility of the Irlen lenses, Irlen's hypotheses, and the scotopic sensitivity syndrome. The analysis includes a review of 13 pro and con research papers. Of special interest is the dichotomy which developed between researchers who were Irlen participants and the professional and scientific community who required less disputable evidence. Even the former, however,_ **failed to find scientific support for Irlen's concept of dysfunction** _in the discharge rate of the retinal receptor cells. Furthermore, in the absence of any evidence that it is a separate and distinct entity, it appears that the scotopic sensitivity syndrome is, in fact, a symptom complex which results primarily from various refractive, binocular, and accommodative disorders._ **Some of the papers which support Irlen's hypotheses provide reason to believe that there is a strong placebo effect**.

> _Interestingly they note that it isn't an optometry or medicine thing, but more of a cognitive thing:_

That could be — the placebo effect is a purely cognitive effect. In fact, it's reasonable that someone might "spin terminology" from "placebo effect" _(which is of negative connotation)_ to "cognitive effect" _(which sounds more, oh I dunno, "sciency")._

I can tell you that even now in 2022, the neurology of visual perception is an under-researched area, and that is party why I've devoted my research activities to this field for the last several years.


> _Whether or how that is covered by WCAG is another question, and it does seem to be something best dealt with on the user-agent end._

Part of the controversy is that the adherents want you to buy their tinted glasses.

IMO things like this are useful to mention in a "general understanding of impairments" but not at a normative level.

**Comment by bruce-usab** [comment]
Can we accept as a fact that that many people have difficulty with pure black text on pure white background?

I just am not convinced that this is something that WCAG -- being content oriented -- can hope to fix.  In my view, it is a problem for operating systems and platforms to address.

The first computers did not try to emulate ink print on paper, but that is where we are now.  Black-on-white is the default, and not just for web browsing.  Probably WCAG can help highlight the barrier, but I don't see how something like "too much contrast" could be anything stricter than AAA.

**Comment by Myndex** [comment]
Hi Bruce @bruce-usab 


> _Can we accept as a fact that that many people have difficulty with pure black text on pure white background?_

Definitely, and I'm one of them.

A project I am working on is "The Paper Reading Experience". — when reading on paper, we are seeing difdfuse light that is typically 80% of the ambient which defines out adaptation level.

The peak white on a display/device tends to be set such that it is the equivelent of 102% ot 120% compared to paper.

But it's not as simple as just lowering the screen brightness—we need to "trick" the brain into adapting to one level, and accepting the area behind text at the right relative luminance. I'm working oin some techniques to make that happen, which is part of the live experiments at the Myndex.com site. 


> _I just am not convinced that this is something that WCAG -- being content oriented -- can hope to fix. In my view, it is a problem for operating systems and platforms to address._

Perhaps, and there are some solutions in that direction, but I am finding "design guidelines" that (seem to) help create the paper reading experience on a standard display. More work is needed. Here is [one experimental example](https://www.myndex.com/WEB/GradientsPartTwo) (not the content, just the page colors).


> _The first computers did not try to emulate ink print on paper, but that is where we are now. Black-on-white is the default, and not just for web browsing. Probably WCAG can help highlight the barrier, but I don't see how something like "too much contrast" could be anything stricter than AAA._

There *is* a "too much contrast" part of APCA (see the [SAPC](https://www.myndex.com/SAPC/) tool for the new use case examples, and it includes a max contrast line).

BUT: if one is to have a max contrast spec, then one also needs a perceptually uniform math method for the visual range. And we already know what my next sentence is, so....
